apps/broadcast-app-indexer/src/api/apps/[appId]/channels/[channelId]/routes.ts

```diff
@@ -0,0 +1,12 @@
+import type { OpenAPIHono } from "@hono/zod-openapi";
+import { channelUnsubscribePOST } from "./unsubscribe/post";
```

Comment:
the naming convention is a bit odd, it doesn't follow verbNonce format and is not camel cased, where do we borrow it from?

Reply:
it is just convention I used here, there is no background except it better matches the actual api path